src/plugins/discover/public/application/components/sidebar/discover_sidebar.tsx

```diff
@@ -174,6 +177,30 @@ export function DiscoverSidebar({
     return result;
   }, [fields]);
 
+  const multiFields = useMemo(() => {
+    if (!useNewFieldsApi || !fields) {
+      return undefined;
+    }
+    const map = new Map();
```

Comment:
:art: (In case we have this type somewhere else we could also reference that, instad of inlining the interfacehere)

```suggestion
    const map = new Map<string, Array<{ field: IndexPatternField; isSelected: boolean }>>();
```